### ISSUE TITLE
Can we add Loading Gif or Message when Loading the list view or expanding the Tree node?

### ISSUE THREAD

**Issue by praveenlobo7** [ygoe/MultiSelectTreeView#37]
Can we add Loading Gif or Message when Loading the list view or expanding the Tree node?

**Comment by ygoe** [comment]
Yes, we can! :-) This is the basic lazy loading pattern. You first add a dummy child under each tree node you want to expand. When the node is expanded (bind that property to your view model and react on it changed), that node becomes visible. This is your loading message. Then you start loading the children asynchronously, and when that's done, you replace the node children with what you've found. The actual children them become visible. Of course you also need to remove the dummy loading node first.

**Comment by praveenlobo7** [comment]
Thank you for information.
Is there any event that can be used when we expand the node?
